### PR TITLE
Ts drag and drop group

### PR DESCRIPTION
**Pull request by Niknikson**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "shortid": "^2.2.15"
   },
   "scripts": {
-    "start": "SET REACT_APP_API_BASE_URL=http://softservedemo.herokuapp.com/ & react-scripts start",
+    "start": "react-scripts start",
```
Comment (ViktoriiaKosovan) [inline frontend/package.json]: use correct package.json, it is not correct

File: frontend/src/components/AddGroupForm/AddGroupForm.js [hunk 2]
```diff
@@ -25,21 +28,35 @@ export const AddGroup = (props) => {
...
         pristine,
         invalid,
+        groups,
         group,
     } = props;
     const { t } = useTranslation('formElements');
 
+    const removeCurrentGroup = () => groups.filter(el => el.id !== group.id);
+
+    const groupsForAfterId = group.id ? removeCurrentGroup() : groups;
```
Comment (ViktoriiaKosovan) [inline frontend/src/components/AddGroupForm/AddGroupForm.js]: Could you use clearer name?

**Review by ViktoriiaKosovan — APPROVED** [review]

File: frontend/src/components/AddGroupForm/AddGroupForm.js (hunk 2 above)
Comment (Niknikson) [reply frontend/src/components/AddGroupForm/AddGroupForm.js]: done

File: frontend/package.json (hunk 1 above)
Comment (Niknikson) [reply frontend/package.json]: done

**Review by HiroFive — APPROVED** [review]

**Review by Yashchemskyi-Oleksandr — APPROVED** [review]

**Review by andrii-kuzmichov — APPROVED** [review]

**Pull request merged**